MaxText/inference_microbenchmark.py

```diff
@@ -27,10 +28,29 @@
 import maxtext_utils
 import pyconfig
 
-
 _WARMUP_ITERS = 2
 
 
+def debug_kv_cache(kv_cache):
+  singler_kv_cache = kv_cache["cache"]["decoder"]["layers_0"]["self_attention"]["AttentionOp_0"]
+  for cache_key in singler_kv_cache.keys():
+    cache_element = singler_kv_cache[cache_key]
+    print(f"{cache_key}:")
```

Comment:
Nit: Would be helpful to print out what the variable name is. You can do this in f-strings by adding an `=` like this. `print(f"{cache_key=}")`

Reply:
Done